docs/user/communication.rst

```diff
@@ -100,6 +100,238 @@ for multiple links.
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+| Save for later                                       |                                      | |AVAILABLE|        | |UNAVAILABLE|       | |PLANNED|           |
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+| Forward Messages                                     |                                      | |WIP|              | |UNAVAILABLE|       | |PLANNED|           |
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+| Resolve Messages                                     | At least tutor and authors           | |AVAILABLE|        | |AVAILABLE|         | |AVAILABLE|         |
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+| Post action bar (thread view)                        |                                      ||NOT PLANNED|       | |AVAILABLE|         | |WIP|               |
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+|                                                                                                                                                              |
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+| **Markdown Textfield Options**                                                                                                                               |
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+| Tag other users                                      |                                      | |AVAILABLE|        | |AVAILABLE|         | |AVAILABLE|         |
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+| Tag channels, lectures and exercises                 |                                      | |AVAILABLE|        | |AVAILABLE|         | |AVAILABLE|         |
++------------------------------------------------------+--------------------------------------+--------------------+---------------------+---------------------+
+| Tag FAQ                                              |                                      | |WIP|              | |WIP|               | |PLANNED|           |
```

Comment:
```suggestion
| Tag FAQ                                              |                                      | |AVAILABLE|              | |WIP|               | |PLANNED|           |
```

As far as i know this feature is already available

Reply:
Imo, it is more referencing channels/lectures/exercises, not taging them. But i dont have a strong opinion on this one

Reply:
I think it doesn't really matter. Referencing sounds better though :)